### PR TITLE
Providing Unformatted JSON Documents

### PR DESCRIPTION
**Pull request by MaxwellKendall**
**Five things:**

1. Removed a `"` in the belgic confession yaml.
2. Re-organized code to reflect anticipation of other data types (or perhaps you only want this in YAML?).
3. Adds script to convert yaml to JSON.
4. removes `/n` and `/` characters from JSON.
5. Adds `.gitignore` and `.prettierrc`

**Rationale:**

Unformatted JSON may be preferable for different consumers. As someone planning to use this data to build a performant, modern static application, I don't want  formatted JSON. The data layer should not handle formatting. That is why we have `CSS`.

That being said, in some implementations, perhaps w/ `reformedstandards.org` for instance, it might be useful to have pre-formatted in JSON.

Using this data myself, I expect to find small changes like `1` above so I'm happy to help with that moving forward.

**Question**
It was very easy to convert `YAML` to `JSON`. My experience with `YAML` has been limited to infrastructure as code. It's interesting to see it here in this form. What was the reason for using this format for the data?

I know we can get this data in JSON through the api, `https://reformedstandards.com/westminster/wcf.json`. Is there a back end normalizing data in YAML?

Again, this is really exciting and impressive so thank you very much for open-sourcing this data.

### CODE REVIEW

**Comment by honza** [comment]
Thank you for your contribution! We're beyond excited that you take an interest in the project.

I'm happy to accept the change to the Belgic Confession. 

However, the other stuff doesn't fit with the spirit of the project.

YAML was carefully chosen as the best trade-off in terms of readability by humans and computers. Many of these documents start as bundles of ugly mess scraped from the Internet, and then are refined and edited by hand. YAML is nice in this aspect because it allows you to see paragraphs easily; JSON not so much. The other end of the spectrum is XML. But most people won't interact with XML because it's painful. YAML is supported by virtually every programming language. 

I'm hesitant to maintain multiple versions of each document because a) it makes it tricky to see which one is canonical, and b) because it's trivial to convert the YAML data to whatever format you want.

The way that the Reformed Standards website is made is somewhat unrelated to this repository. There is no API, and no backend. It's a completely static website. We simply read the YAML files in, and spit out some HTML.

**Comment by MaxwellKendall** [comment]
@honza understood! I applaud the choice of YAML. It was almost entirely painless to convert to JSON. Well done!

My plan is to make a similar website to `reformedstandards` using the Static Site Generator Gatsby or perhaps next.js and to use JSON documents, from the YAML, as the source. As mostly a front end developer, I'm really excited and share the exact goal of `reformedstandards.org` and have a very similar domain name `confessionalchristianity.com`. I'd rather not split off to my own project, but perhaps we can join forces in some way down the road.

Basically all of the hard work I was planning on doing -- normalizing the confessional documents into a machine readable format -- is done (in spades, with more data than I could've completed in years and years of work!) thanks to you open-sourcing this repo. I'm exceedingly grateful and over the moon about it.

Moving forward, I'll only push changes to the YAML files as I come to find errors in my `JSON`.

Thank you again for all your hard work! Would love to learn more about how you came up with all this.
